packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -190,35 +242,57 @@ export function LeftRail() {
         </Content>
         <Content>
           <NavList>
-            {enabledPrimaryPaths.map(({ Icon, ...path }) => (
-              <NavListItem key={path.value}>
-                <NavLink
-                  active={path.value === state.currentPath}
-                  href={resolveRoute(path.value)}
-                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-                  isBold
-                  aria-label={
-                    path.value === state.currentPath
-                      ? sprintf(
-                          /* translators: %s: the current page, for example "My Stories". */
-                          __('%s (active view)', 'web-stories'),
-                          path.label
-                        )
-                      : path.label
-                  }
-                  {...(path.isExternal && {
-                    rel: 'noreferrer',
-                    target: '_blank',
-                    onClick: (evt) => onExternalLinkClick(evt, path),
-                  })}
-                >
-                  {Icon && <Icon width="22px" />}
-                  <PathName as="span" isBold>
-                    {path.label}
-                  </PathName>
-                </NavLink>
-              </NavListItem>
-            ))}
+            {enabledPrimaryPaths.map(({ Icon, ...path }) => {
+              const isNotificationBubbleEnabled =
+                path.value === APP_ROUTES.TEMPLATES_GALLERY;
+              const appendNewBadgeToLable = (label) =>
+                isNotificationBubbleEnabled
+                  ? sprintf(
+                      /* translators: 1: current page. 2: number of new items. */
+                      __('%1$s (%2$s new)', 'web-stories'),
+                      label,
+                      numNewTemplates
+                    )
+                  : label;
+              return (
+                <NavListItem key={path.value}>
+                  <NavLink
+                    active={path.value === state.currentPath}
+                    href={resolveRoute(path.value)}
+                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                    isBold
+                    aria-label={appendNewBadgeToLable(
+                      path.value === state.currentPath
+                        ? sprintf(
+                            /* translators: %s: the current page, for example "My Stories". */
+                            __('%s (active view)', 'web-stories'),
+                            path.label
+                          )
+                        : path.label
+                    )}
+                    {...(path.isExternal && {
+                      rel: 'noreferrer',
+                      target: '_blank',
+                      onClick: (evt) => onExternalLinkClick(evt, path),
+                    })}
+                  >
+                    <IconWrap>
+                      {Icon && <Icon width="22px" />}
```

Comment:
Hardcoded width?

Reply:
This wasn't an update on my end. part of the existing code. regardless, I think it gets the job done and isn't doing any harm in this context